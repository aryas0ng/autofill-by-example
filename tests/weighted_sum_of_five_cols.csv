1,9,0,2,3,42
2,8,5,1,3,52
2,7,4,0,34,198
3,6,3,2,4,52
10,5,2,1,4,50
5,4,1,0,6,46
4,3,0,9,4,
19,2,2,8,1,
2,1,1,7,14,
1,0,0,6,5,
9,4,6,5,1,
7,3,5,4,1,
4,2,4,3,1,
3,1,3,2,1,
12,0,2,1,11,
4,1,1,0,12,
1,0,0,2,16,
26,5,3,1,16,
8,4,2,0,17,
3,3,1,3,2,
3,2,0,2,2,
8,1,4,1,2,
4,0,3,0,20,
6,9,2,5,20,
5,8,1,4,27,
4,7,0,2,4,
3,6,6,1,4,
2,5,5,0,5,
1,4,4,2,54,
0,3,3,1,6,
3,2,2,3,6,
2,1,1,3,7,
1,0,0,4,8,
0,4,3,46,8,
6,3,2,5,8,
5,2,1,6,15,