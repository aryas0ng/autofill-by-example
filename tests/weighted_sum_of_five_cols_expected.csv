1,9,0,2,3,42
2,8,5,1,3,52
2,7,4,0,34,198
3,6,3,2,4,52
10,5,2,1,4,50
5,4,1,0,6,46
4,3,0,9,4,66
19,2,2,8,1,66
2,1,1,7,14,105
1,0,0,6,5,50
9,4,6,5,1,60
7,3,5,4,1,49
4,2,4,3,1,37
3,1,3,2,1,27
12,0,2,1,11,77
4,1,1,0,12,69
1,0,0,2,16,89
26,5,3,1,16,129
8,4,2,0,17,107
3,3,1,3,2,34
3,2,0,2,2,25
8,1,4,1,2,36
4,0,3,0,20,113
6,9,2,5,20,150
5,8,1,4,27,175
4,7,0,2,4,46
3,6,6,1,4,57
2,5,5,0,5,52
1,4,4,2,54,299
0,3,3,1,6,49
3,2,2,3,6,55
2,1,1,3,7,54
1,0,0,4,8,57
0,4,3,46,8,241
6,3,2,5,8,78
5,2,1,6,15,111